Nos/Controller/PersistenceController.swift

```diff
@@ -144,6 +102,7 @@ class PersistenceController {
         }
     }
     
+#if DEBUG
```

Comment:
I'd love it if we moved the DEBUG code to an extension. I'm totally fine with it living inside this file -- I just think it would be a little nicer in an extension. We have similar extensions for `SettingsView` and `DefaultFeatureFlags`.

Reply:
Yeah, that's nicer. Thanks!